content/2022-05-19-leave-the-yaml-at-home.md

```diff
@@ -0,0 +1,185 @@
+[__projects__](https://docs.brigade.sh/topics/project-developers/projects/).
+Projects pair __event subscriptions__ with configuration for an event handler
+called a __worker__. These project definitions are often quite concise and
+are typically the only YAML you write when using Brigade.
+
+To demonstrate, let's look at a _real_ project definition. This happens to be
+Brigade's _own_ project definition!
+
+```yaml
+# yaml-language-server: $schema=https://schemas.brigade.sh/schemas-v2/project.json
+apiVersion: brigade.sh/v2
+kind: Project
+metadata:
+  id: brigade
+description: Brigade built with Brigade!
+spec:
+  eventSubscriptions:
+  - source: brigade.sh/github
+    qualifiers:
+      repo: brigadecore/brigade
+    types:
+    - ci:pipeline_requested
+    - ci:job_requested
+    - cd:pipeline_requested
+  workerTemplate:
+    git:
+      cloneURL: https://github.com/brigadecore/brigade.git
+```
+
+While the above closely resembles a Kubernetes manifest, it is not. Let's pick
+it apart:
+
+* If your text editor or IDE integrates with the YAML language server, the
+  comment on the first line will allow your editor or IDE to validate your
+  project definition and offer context help. (I use
+  [VS Code](https://code.visualstudio.com/) with
+  [YAML Language Support by Red Hat](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml).)
+
+* As with a Kubernetes manifest, we move on to denote what type of resource this
+  YAML describes using the `apiVersion` and `kind` fields, and the `metadata.id`
+  field assigns an immutable identifier to our project. The `description` field
+  is probably self-explanatory.
+
+* The `spec` portion of the file does exactly what we said earlier that projects
+  do -- it pairs event subscriptions with worker configuration.
+
+    * The `eventSubscriptions` field uses `source`, `types`, and `qualifiers` to
+      describe events of interest. Event gateways and other event sources
+      document what sort of events they emit and exactly how to subscribe to
+      them. Here, we're interested in three specific event types that originate
+      from the Brigade GitHub Gateway (identified by `source:
+      brigade.sh/github`)
+      _provided_ the gateway qualified the event as being related to the
+      [`brigadecore/brigade`](https://github.com/brigadecore/brigade) repository
+      on GitHub. (For reference, all events available from that gateway are
+      described
+      [here](https://github.com/brigadecore/brigade-github-gateway/blob/main/docs/EVENT_REFERENCE.md).)
+
+    * The `workerTemplate` portion of the file describes the worker that should
+      handle the events. In this simple case, it only uses the `git.cloneURL`
+      field to indicate where project source -- including the script we'd like
+      to run -- can be obtained.
+
+This is all the YAML you write! (And, actually, most of this can be generated
+for you with the `brig init` command!)
```

Comment:
I think the parentheses should go. That seems like a really important point that maybe even deserves a little bit more prominence.

Reply:
sgtm